packages/react/src/components/OverflowMenu/OverflowMenu.js

```diff
@@ -493,27 +492,29 @@ class OverflowMenu extends Component {
     );
 
     const menuBody = (
-      <ul className={overflowMenuOptionsClasses} tabIndex="-1" role="menu">
+      <ul
+        className={overflowMenuOptionsClasses}
+        tabIndex="-1"
+        role="menu"
+        aria-label="Menu">
```

Comment:
```suggestion
        aria-label={ariaLabel}>
```

Reply:
`{ariaLabel}` re-added, `Menu` added to `defaultProps` on line 209 and line 23 in `OverflowMenu-story.js`